core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -406,10 +407,18 @@ public static Duration getDuration(PropertyKey key) {
   }
 
   /**
-   * @return a view of the internal {@link Properties} of as an immutable map
+   * @return a view of the properties(include default properties) of as an immutable map
    */
   public static Map<String, String> toMap() {
-    return Collections.unmodifiableMap(PROPERTIES);
+    Map<String, String> map = new HashMap<>(PROPERTIES);
```

Comment:
if we want a view we should use `Collections.unmodifiableMap`. If we want an immutable copy, we should use `ImmutableMap.builder`.

Reply:
I don't think here we want an immutable copy. Collections.unmodifiableMap maybe just used to turn values into resolved correspondings.